components/layouts/DocsLayout.js

```diff
@@ -0,0 +1,161 @@
+import getIcon from "../Icon";
+
+export default function DocsLayout({ children }) {
+  function classNames(...classes) {
+    return classes.filter(Boolean).join(" ");
+  }
+
+  const displayIcon = (page) => {
+    const Icon = getIcon(page.icon);
+    return (
+      <Icon
+        className={classNames(
+          page.current
+            ? "text-indigo-500"
+            : "text-gray-400 group-hover:text-gray-500",
+          "mr-3 flex-shrink-0 h-6 w-6"
+        )}
+        aria-hidden="true"
+      />
+    );
+  };
+  const pages = [
+    {
+      name: "Docs home",
+      path: "/docs",
+      icon: "FaHome",
+    },
+    {
+      name: "Quickstart",
+      path: "/docs/quickstart",
```

Comment:
I think the `path` property here isn't correct for some pages. They need to reflect the folder-structure in `pages`. I'm pretty sure you're aware of this though 👍

Reply:
Thanks Cahl, good spot 👍  Yep we started refactoring yesterday the file structure, but didn't get time to update the navigation yet 🤦‍♂️ 

We are hoping to have this PR ready this coming week 🎉

Reply:
I can make that change to the layout, if it helps you 👍

Reply:
That would be great, if you have time 👍